Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -172,7 +172,7 @@ defineSuite([
 
     it('rejects readyPromise with invalid tileset.json', function() {
         var tileset = scene.primitives.add(new Cesium3DTileset({
-            url : 'invalid'
+            url : 'invalid.json'
```

Comment:
After sweeping over the code this seems to be the common practice. Just as an example, search for `fail('should not resolve');`

Is it worth opening an issue for this?